test/acceptance/cucumber.js

```diff
@@ -0,0 +1,9 @@
+
+module.exports = {
+  default: {
+    publishQuiet: true,
+    require: ['step_definitions/**/*.js'],
+    parallel: 1,
+    format: ['html:reports/cucumber-report.html']
```

Comment:
Suggest outputting to existing `test-output` folder to be consistent with all other test outputs.  Also avoids permission issues in volume mapping.